src/Distribution/Nixpkgs/Haskell/FromCabal/License.hs

```diff
@@ -165,5 +165,5 @@ fromSPDXLicense (SPDX.License expr) =
 
 isFreeLicense :: Distribution.Nixpkgs.License.License -> Bool
 isFreeLicense (Known "lib.licenses.unfree") = False
-isFreeLicense (Unknown Nothing)             = False
+isFreeLicense (Unknown Nothing)             = True
```

Comment:
Could you leave a comment on this line explaining the reasoning for this?

Probably just copying some of the PR comment body here would be fine.

If someone is looking through this code, I imagine they might be surprised that unknown licenses are marked as free.

Other than that, this LGTM!